### PR TITLE
Jenkins: Don't write AWS creds to disk

### PR DESCRIPTION
**Pull request by zmc**
There's much less of a risk of exposing credentials via the workspace if we just never write them to disk.

### CODE REVIEW

**Comment by zmc** [comment]
[Updates for ocs-ci Jenkins job](https://trello.com/c/rn07ZZ6B/13-updates-for-ocs-ci-jenkins-job)

File: .functional_ci_setup.py [hunk 1]
```diff
@@ -68,7 +89,16 @@ def get_ocsci_conf():
         ),
     )
     # Apply image configuration if present
-    for image_type in ['rook', 'ceph']:
+    image_types = [
+        'rook',
+        'ceph',
+        'ceph_csi',
+        'rook_csi_registrar',
```
[the remark this answers is not in the document]
Comment (zmc) [reply .functional_ci_setup.py]: I'm actually just using the values from that yaml file - it's just that they're embedded as defaults in the JJB definition

File: .functional_ci_setup.py [hunk 2]
```diff
@@ -8,6 +9,26 @@
 from configparser import ConfigParser
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--skip-aws', '-a',
```
[the remark this answers is not in the document]
Comment (zmc) [reply .functional_ci_setup.py]: That's perfectly reasonable - I'm not actually using the short option at all. I can remove it.

[the remark this answers is not in the document]
Comment (zmc) [reply .functional_ci_setup.py]: @petr-balogh if you wouldn't mind re-approving, I've just repushed with that as the only change.

[the remark this answers is not in the document]
Comment (petr-balogh) [reply .functional_ci_setup.py]: @zmc I thought you will delete those one char parameters also for other two parameters below ;)

[the remark this answers is not in the document]
Comment (zmc) [reply .functional_ci_setup.py]: @petr-balogh whoops, that was an oversight. Done!

**Comment by zmc** [comment]
@andrewschoen mind taking a look?

**Comment by zmc** [comment]
Thanks @petr-balogh!

**Pull request merged**